### PR TITLE
Swift 3

### PR DESCRIPTION
**Pull request by danielablg**

### CODE REVIEW

File: TABScrollingContentView.podspec [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Pod::Spec.new do |s|
   s.name         = 'TABScrollingContentView'
-  s.version      = '1.2.1'
-  s.platform	 = :ios, '8.0'
+  s.version      = '2.0.0'
+  s.platform	 = :ios, '10.0'
```
Comment (lucianomarisi) [inline TABScrollingContentView.podspec]: why has this changed to 10?

**Review by lucianomarisi — CHANGES_REQUESTED** [review]
could the Gemfile and Gemfile.lock files go

**Review by lucianomarisi — CHANGES_REQUESTED** [review]
could the Gemfile and Gemfile.lock files go?

File: TABScrollingContentView.podspec (hunk 1 above)
Comment (danielablg) [reply TABScrollingContentView.podspec]: I have reversed that in the latest commit

File: README.md [hunk 2]
```diff
@@ -38,6 +38,6 @@ You can alternatively add subviews yourself. If doing so, you should add the sub
 There are also the following methods that can be used to add vertical or horizontal subviews to any type of view.
 
 ```
-addVerticalSubviews(_:withMargins:verticalSpacing:)
-addHorizontalSubviews(_:withMargins:horizontalSpacing:)
+addVertical(subviews:withMargins:verticalSpacing:)
+addHorizontal(subviews:withMargins:horizontalSpacing:)
```
Comment (samdods) [inline README.md:42]: should be `add(horizontalSubviews:`

Comment (samdods) [inline README.md]: should be `add(horizontalSubviews:`

Comment (samdods) [inline README.md]: should be `add(horizontalSubviews:`

File: Sources/ScrollingContentView.swift [hunk 3]
```diff
@@ -105,51 +105,51 @@ public class VerticalScrollingContentView: UIView {
...
-    addConstraintToView(self, targetAttribute: leadingAttribute, fromContentViewAttribute: leadingAttribute)
-    addConstraintToView(self, targetAttribute: trailingAttribute, fromContentViewAttribute: trailingAttribute)
+    addConstraint(toView: self, targetAttribute: leadingAttribute, fromContentViewAttribute: leadingAttribute)
+    addConstraint(toView: self, targetAttribute: trailingAttribute, fromContentViewAttribute: trailingAttribute)
   }
   
-  private func addConstraintToView(targetView: UIView, targetAttribute: NSLayoutAttribute, fromContentViewAttribute sourceAttribute: NSLayoutAttribute) {
-    let constraint = NSLayoutConstraint(item: contentView, attribute: sourceAttribute, relatedBy: NSLayoutRelation.Equal, toItem: targetView, attribute: targetAttribute, multiplier: 1, constant: 0)
+  private func addConstraint(toView targetView: UIView, targetAttribute: NSLayoutAttribute, fromContentViewAttribute sourceAttribute: NSLayoutAttribute) {
+    let constraint = NSLayoutConstraint(item: contentView, attribute: sourceAttribute, relatedBy: NSLayoutRelation.equal, toItem: targetView, attribute: targetAttribute, multiplier: 1, constant: 0)
```
Comment (samdods) [inline Sources/ScrollingContentView.swift:134]: could be `addConstraint(to:`

File: Sources/ScrollingContentView.swift [hunk 4]
```diff
@@ -105,51 +105,51 @@ public class VerticalScrollingContentView: UIView {
...
   }
   
   // MARK: Static helpers
   
   private static func newScrollView() -> UIScrollView {
-    let scrollView = UIScrollView(frame: CGRectZero)
-    scrollView.backgroundColor = UIColor.clearColor()
-    scrollView.autoresizingMask = [.FlexibleWidth, .FlexibleHeight]
+    let scrollView = UIScrollView(frame: CGRect.zero)
+    scrollView.backgroundColor = UIColor.clear
```
Comment (samdods) [inline Sources/ScrollingContentView.swift]: no need to change but just fyi you could use `= .clear` and `frame: .zero` for simplicity

File: Sources/UIView+Subviews.swift [hunk 5]
```diff
@@ -30,45 +30,45 @@ public extension UIView {
...
   // MARK: - Vertical
   
-  public func addVerticalSubviews(subviews: [UIView], withMargins margins: EdgeMargins = EdgeMargins(), verticalSpacing: Double = 0) {
+  public func addVertical(subviews: [UIView], withMargins margins: EdgeMargins = EdgeMargins(), verticalSpacing: Double = 0) {
     var previousView: UIView? = nil
     for subview in subviews {
       subview.translatesAutoresizingMaskIntoConstraints = false
       self.addSubview(subview)
-      addConstraintsFromView(subview, previousView: previousView, margins: margins, verticalSpacing: verticalSpacing)
+      addConstraints(fromSubiew: subview, previousView: previousView, margins: margins, verticalSpacing: verticalSpacing)
```
Comment (samdods) [inline Sources/UIView+Subviews.swift:38]: could be `addConstraints(from:`

**Review by lucianomarisi — APPROVED** [review]

**Pull request merged**